documentation/src/pages/oauth/providers/github.md

```diff
@@ -22,18 +22,20 @@ const github: (
 		clientId: string;
 		clientSecret: string;
 		scope?: string[];
+		state?: boolean;
```

Comment:
I also prefer to have state configurable in `OAuthConfig` but can’t for the life of me make it type safe. It might better if state is configurable in `getAuthorizationUrl()`. (Re-adding the comment since it made no sense with the typo)